regression/book-examples/CMakeLists.txt

```diff
@@ -30,11 +30,11 @@ add_test_pl_profile(
     "CORE"
 )
 
-# If `-I` (include flag) is passed, test.pl will run only the tests matching the label following it.
+# If `-I` (include flag) is passed, test.pl will exclude the tests matching the label following it.
```

Comment:
Comment is now wrong.

Reply:
Fixed. The explanation in the comment is now updated in all 3 places.